src/main/java/friday/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,71 @@
+package friday.logic.parser;
+
+import static friday.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static friday.logic.parser.CliSyntax.ORDER_ASCENDING;
+import static friday.logic.parser.CliSyntax.PREFIX_CONSULTATION;
+import static friday.logic.parser.CliSyntax.PREFIX_MASTERYCHECK;
+import static friday.logic.parser.CliSyntax.PREFIX_NAME;
+import static friday.logic.parser.CliSyntax.PREFIX_TELEGRAMHANDLE;
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import friday.logic.commands.SortCommand;
+import friday.logic.parser.exceptions.ParseException;
+import friday.model.student.Student;
+
+/**
+ * Parses input arguments and creates a new EditCommand object
+ */
+public class SortCommandParser implements Parser<SortCommand> {
```

Comment:
Javadoc header does not seem accurate, would creates a new SortCommand object be better?